app/models/invitation_manager.rb

```diff
@@ -1,111 +1,59 @@
 class InvitationManager
   def send_workshop_emails(workshop, audience)
     return 'The workshop is not invitable' unless workshop.invitable?
-
-    if audience == 'students'
-      invite_students workshop
-      return
-    end
-
-    if audience == 'coaches'
-      invite_coaches workshop
-      return
-    end
-
-    invite_students workshop
-    invite_coaches workshop
+    invite_students_to_workshop(workshop) unless audience.eql?('coaches')
```

Comment:
Never see `eql?` used like this, is this a convention established for this codebase?

Reply:
@barisbalic I don't believe there is any convention established for the codebase. `eql?` is quite common practise in rails project.